replay_build_scripts/syncRepo.js

```diff
@@ -0,0 +1,16 @@
+const { spawnChecked } = require("../buildLinux");
```

Comment:
If you want to consolidate this file with "common" (or whatever you move spawnChecked into,) I'd be fine with that.  My principle is generally to keep utility/agnostic function together.